src/content/roadmap/_index.en.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "Roadmap"
+date = 2020-02-19T20:24:34+01:00
+weight = 100
+pre = "<b>5. </b>"
++++
+
+This a preliminary community roadmap, it's not written in stone, but it can serve
+as a guideline about what's ahead.
+
+Please see details of previous releases [here](../releases)
+
+### v0.1.1
+
+* Stabilization of new lighthouse related features and deployment
+* Bugfixes
+
+### v0.2.0
```

Comment:
what about Armada? Will be having a specific release for it?

Reply:
Yes, but that's more internal tool for CI, we don't need to publicize that to end users.

Reply:
* Documentation & website.